app/src/main/res/layout/help_card.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/helpCardBackground">
+
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/help_card"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:cardElevation="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:shapeAppearanceOverlay="@style/CardSquareBottomCorners"
+        app:strokeColor="@color/cardBorder"
+        app:strokeWidth="1dp">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:layout_marginHorizontal="@dimen/horizonal_margin"
+            android:paddingTop="20dp"
+            android:paddingBottom="12dp">
+
+            <TextView
+                android:id="@+id/help_header"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                tools:text="@string/todays_score_header"
+                style="@style/MainHeader" />
```

Comment:
per figma I think there's a margin of about 16dp between the header and the help text here, and 60dp margin below the bottom of the button. Otherwise looks great.